app/views/categories/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="app">
+  <div class="category__content">  
+    <header class="header">
+      <nav class="nav">
+        <span></span>
+        <span>CATEGORIES</span>
+        <%= link_to "Log out", destroy_user_session_path, class: "action", method: :delete %>
+      </nav>
+    </header>
+    <div class="flash__message">
+      <p class="notice"><%= notice %></p>
+      <p class="alert"><%= alert %></p>
+    </div>
+    <ul class="category__list">
+      <% if @groups.empty? %>
+        <div class="empty">
+          No categories for now
+        </div>
+      <% else %>
+        <% @groups.each do |group| %>
+          <%= render partial: 'group_card', locals: { group: group } %>
+        <% end %>
+      <% end %>
+    </ul>
+  </div>
+
+  <div class="cta__container">
+    <%= link_to "ADD A NEW CATEGORY", new_category_path, class: "add__btn" %>
```

Comment:
- [ ] On adding category it throws an error and then I cannot move forward. 
![Capture](https://user-images.githubusercontent.com/58504655/142926397-af43da31-6b16-48f1-bcbb-dadbb689e5fd.PNG)

Reply:
Hi @arslanbisharat 
Thank you for your time and effort on the review
I guess you tried the deployed link. I just added a new category it is working as it should. kindly find below a snapshot of the working one
![Screen Shot 2021-11-22 at 21 06 06](https://user-images.githubusercontent.com/55432600/142928009-9bef2a88-c7f8-44ac-9c18-a8d7d3adb944.png)